src/design-tokens/colors.yml

```diff
@@ -1,140 +1,140 @@
 aliases:
-  blue_lighter: "#8abfff"
-  blue_light: "#3d84F5"
-  blue: "#215cca"
-  blue_dark: "#183889"
-  blue_darker: "#0E1C43"
-  green_lighter: "#71f4ae"
-  green_light: "#2cdd90"
-  green: "#158466"
-  green_dark: "#105c52"
-  green_darker: "#0c3538"
-  purple_lighter: "#e98efa"
-  purple_light: "#d250f7"
-  purple: "#950cde"
-  purple_dark: "#560a9b"
-  purple_darker: "#2a085e"
-  fuchsia_lighter: "#ff80ca"
-  fuchsia_light: "#f240ae"
-  fuchsia: "#d9118f"
-  fuchsia_dark: "#950f7d"
-  fuchsia_darker: "#5c0a5c"
-  orange_lighter: "#ffad5c"
-  orange_light: "#fa8f47"
-  orange: "#f27041"
-  orange_dark: "#aa3f24"
-  orange_darker: "#622018"
-  gray_lighter: "#dee7ec"
-  gray_light: "#a5b9c6"
-  gray: "#738da0"
-  gray_dark: "#485968"
-  gray_darker: "#252b32"
-  black_transparent_85: "rgba(0, 0, 0, 0.85)"
-  white_transparent_85: "rgba(255, 255, 255, 0.85)"
-  white: "#fff"
+  blue_lighter: '#8abfff'
```

Comment:
My code editor made this swap automatically. I'm guessing it's based on Prettier or linting rules? Happy to revert if preferred. The only actual changes are to `gray_light` and `gray_lighter`

Reply:
Yeah I think it is because of prettier. I think this change is fine 👍